src/test/java/org/kiwiproject/registry/consul/client/ConsulRegistryClientTest.java

```diff
@@ -48,7 +57,15 @@ void setUp() {
                         .tags(List.of("service-type:default", "category:CORE"))
                         .meta(Map.of(
                                 "adminPort", "8081",
-                                "scheme", "http"
+                                "scheme", "http",
+                                "serviceUpTimestamp", Long.toString(now),
```

Comment:
I presume this is a Consul limitation (or, at least a requirement of the Orbiz Consul library) that it requires metadata to be a map of String to String?

Reply:
Nevermind...I see this isn't even our class...